lib/modules/hover.js

```diff
@@ -69,6 +75,7 @@ module.beforeLoad = () => {
 		fadeSpeed: parseFloat(module.options.fadeSpeed.value),
 		width: parseFloat(module.options.width.value),
 		closeOnMouseOut: module.options.closeOnMouseOut.value,
+		closeOnTargetUnmount: module.options.closeOnMouseOut.value,
```

Comment:
I don't see a need to have an option for this.

Reply:
I did that because of the `if` check.

Reply:
We probably don't need that conditional, since it seems useful to apply this everywhere.